packages/synapse-react-client/src/components/QueryWrapperPlotNav/UseRowSet.test.tsx

```diff
@@ -0,0 +1,128 @@
+import { act, renderHook as _renderHook, waitFor } from '@testing-library/react'
+import { createWrapper } from '../../testutils/TestingLibraryUtils'
+import { useRowSet } from './UseRowSet'
+import React from 'react'
+import { server } from '../../mocks/msw/server'
+import { QueryWrapper } from '../QueryWrapper'
+import {
+  mockQueryBundleRequest,
+  mockQueryResultBundle,
+} from '../../mocks/mockFileViewQuery'
+
+import { registerTableQueryResult } from '../../mocks/msw/handlers/tableQueryService'
+
+describe('UseRowSet', () => {
```

Comment:
Tests for the hook that managed getting `RowSet` data and properly paginating it